automatic/transifex-cli/Readme.md

```diff
@@ -0,0 +1,11 @@
+# [<img src="https://cdn.jsdelivr.net/gh/chocolatey-community/chocolatey-packages@4c2d7c7377b05a3d0a2dd35cfc89ce0c77625051/icons/transifex-client.png" height="48" width="48" /> Transifex Client](https://chocolatey.org/packages/transifex-client)
```

Comment:
```suggestion
# [<img src="https://cdn.jsdelivr.net/gh/chocolatey-community/chocolatey-packages@4c2d7c7377b05a3d0a2dd35cfc89ce0c77625051/icons/transifex-cli.png" height="48" width="48" /> Transifex CLI](https://chocolatey.org/packages/transifex-cli)
```

Please also rename the existing icon for `transifex-client` to `transifex-cli` (assuming it is still the same icon being used by the software, if it isn't, then remove the icon and extract the new icon from the software using `becyicongrabber` or a similar program).

Reply:
The new version of the CLI doesn't actually have any icon as far as I can tell, so I just used the old one for now 🤷‍♀️

Reply:
Also, if I rename this file, won't the link be invalid since it contains a commit hash?

Reply:
That is exactly why we use commit hashes, so it won't be invalid. It would be invalid if we didn't use commit hashes.

Reply:
I believe I've updated this one and the link seems to work, so we should be good.